InAppUtils/InAppUtils.m

```diff
@@ -159,6 +176,18 @@ - (void)paymentQueueRestoreCompletedTransactionsFinished:(SKPaymentQueue *)queue
     [[SKPaymentQueue defaultQueue] restoreCompletedTransactions];
 }
 
+RCT_EXPORT_METHOD(restorePurchasesForUser:(NSString *)username
+                    callback:(RCTResponseSenderBlock)callback)
+{
+    NSString *restoreRequest = @"restoreRequest";
+    _callbacks[RCTKeyForInstance(restoreRequest)] = callback;
+    if(!username) {
+        callback(@[@"no_username"]);
```

Comment:
@RalfNieuwenhuizen can we make it little more descriptive? "username_required" maybe? just trying to ensure that we avoid confusion.